### ISSUE TITLE
Tidy up fragalysis repos in githup - root repo etc.

### ISSUE THREAD

**Issue by phraenquex** [m2ms/fragalysis-frontend#1122]
Intitial draft, for now.

**Comment by phraenquex** [comment]
Would happen after #1077

**Comment by xchem-super** [comment]
Getting started spinning up development of stack, specifically, how to build a container image and the architecture of Fragalysis (how it is put together?).
the target audience is stack developers, this is very much related to concept/idea of dev containers (@alanbchristie and @Waztom to review)

**Comment by phraenquex** [comment]
Create a "fragalysis-root" repo (or similar).

Retire the https://github.com/xchem/dls-fragalysis-stack-kubernetes repo

Adopt @alanbchristie's stack repo.  

And make a nice Read.me file.

**Comment by phraenquex** [comment]
Will happen in the new year- can wait, no functional implications.

**Comment by alanbchristie** [comment]
To deal with the root repo, 'fork' https://github.com/xchem/dls-fragalysis-stack-kubernetes into `xchem`. Once done you can then manage any documentation and Ansible playbooks.

The repo is responsible for all the Ansible playbooks that are executed by the AWX server in the developer and production clusters. Once the repo has been forked edit the **AWX Project** records on the two **AWX** servers. This will ensure that the new repository location is used when downloading the playbooks when the corresponding **Job Templates** run. The two servers are located at: -

- https://awx-xchem.informaticsmatters.org/ (**developer**)
- https://awx-xchem-production.informaticsmatters.org/ (**production**)

When the switchover is complete we can then **Archive** the IM repository (make it _read only_).

> We can fork the repository at any time, the switch over can be done gradually.

**Comment by alanbchristie** [comment]
The origin repo is already forked to `xchem`: -

- https://github.com/xchem/dls-fragalysis-stack-kubernetes

Someone with authority probably needs to: -

1. Synchronise this fork with the upstream (origin) repo (see https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#)
2. Rename the fork to `fragalysis-stack-kubernetes`?
3. Update the ReadTheDocs page so its content is compiled from the fork rather than the origin.

**Comment by phraenquex** [comment]
@Waztom please act on @alanbchristie's request.

**Comment by Waztom** [comment]
@alanbchristie have added you with write permissions. Please let know if you need more access (admin level).

**Comment by alanbchristie** [comment]
I have now archived the Informatics Matters repository (https://github.com/InformaticsMatters/dls-fragalysis-stack-kubernetes). This repo has now been forked to the xchem domain as: -

- https://github.com/xchem/fragalysis-stack-kubernetes
It is used by AWX but I will slowly switch the AWS servers over to this new repository

**Comment by alanbchristie** [comment]
@Waztom As I have resorted to re-forking the original repository, the preceding fork can now be archived, and probably removed?

- https://github.com/xchem/dls-fragalysis-stack-kubernetes

...because this has been replaced by...

- https://github.com/xchem/fragalysis-stack-kubernetes

**Comment by Waztom** [comment]
@alanbchristie one thing, can you please confirm that we will not be loosing code/fork that was specifically applicable to deploying on Diamond's Kubernetes? Diamond/Chris has their own Kubernetes (last I heard, still only available to people accessing from within the Diamond firewall), whilst Fragalysis is deployed on STFC's Kubernetes.

**Comment by alanbchristie** [comment]
AFAIK (and after reviewing the commits) the fork of our (original) repository was only changed by Rachael for three purposes: -

1. Updated the documentation configuration (probably for ReadTheDocs) [10 Jun 2020] **(Now out of date)**
2. Fixed Travis [12 Jun 2020] **(Travis is no longer used)**
3. Adjusted sphinx main documentation [2 Sep 2020] **(Now out of date)**

With regard to deployment to Diamond Kubernetes I don't think any of the playbooks are used ... unless someone has deployed the stack to Diamond without my/our knowledge (which is unlikely).

The playbooks for deployment to STFC's Kubernetes are all contained in this new fork and the two AWX servers (in the **dev** and **prod** clusters) have now been updated to use this new fork as a source.

I have also updated **ReadTheDocs** to generate new pages from the new repository.

Our original repository has now been "archived" (set to _read only_) and is no longer used.

In conclusion: -

- I think the risk to existing processes by removal of the repo is _extremely low_
- Someone need to delete the old ReadTheDocs project

**Comment by phraenquex** [comment]
@Waztom, @mwinokan who of you gets this action?

**Comment by Waztom** [comment]
@Waztom will look into this.

**Comment by Waztom** [comment]
@alanbchristie will ping link/info to delete.

**Comment by alanbchristie** [comment]
I think we can consider this done? Or at least old enough to retire and replace wit something more meaningful?
